cpp/oneapi/dal/algo/jaccard/common.hpp

```diff
@@ -58,23 +58,19 @@ class descriptor : public descriptor_base {
 public:
     using method_t = Method;
 
-    auto &set_row_begin(std::int64_t value) {
-        this->set_row_begin_impl(value);
+    auto& set_row_range(const int64_t& begin, const int64_t& end) {
+        this->set_row_range_impl(begin, end);
         return *this;
     }
 
-    auto &set_row_end(std::int64_t value) {
-        this->set_row_end_impl(value);
+    auto& set_column_range(const int64_t& begin, const int64_t& end) {
+        this->set_column_range_impl(begin, end);
         return *this;
     }
 
-    auto &set_column_begin(std::int64_t value) {
-        this->set_column_begin_impl(value);
-        return *this;
-    }
-
-    auto &set_column_end(std::int64_t value) {
-        this->set_column_end_impl(value);
+    auto& set_block(const std::initializer_list<int64_t>& row_range,
```

Comment:
std::int64_t?
We always use 64 bit integer to represent ranges. It can be fine but to think-should we parametrize it somehow?

Reply:
@andreynv 
Discussed in email that we use int64_t to cover all cases.
Possible option to parameterize it through graph alias.
set_block<vertex_type<graph>>(...); 
With this option we make user to write some input constraint for algorithm descriptor which, in theory, should not be logically connected with input (graph).
Also, API will take more code.

Reply:
this analysis is not required here and now - you will need to analyze as one of the next steps